en/app.ftl

```diff
@@ -1088,3 +1115,7 @@ forwarded-email-footer = Stop email forwarding and manage settings for all alias
 forwarded-email-footer-2 = Stop email forwarding and manage settings for all masks here.
 # This entire text is a link
 forwarded-email-footer-premium-banner = Upgrade to { -brand-name-relay-premium }
+
+## Notifications component
+
+toast-button-close-label = Close notification
```

Comment:
@lloan 

Adding strings after the PR was reviewed and then merging right away is not OK, it defies the point of having mandatory reviews.

In particular, this string should have landed leaving an empty line at the end of the file. Next time someone adds a string after, this one will change :-|